packages/kinvey-datastore2/test/datastore.spec.js

```diff
@@ -0,0 +1,79 @@
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import { DataStore, DataStoreType, collection } from '../src';
+
+describe('DataStoreType', () => {
+  it('should have type Cache', () => {
+    expect(DataStoreType.Cache).to.not.be.undefined;
+  });
+
+  it('should have type Network', () => {
+    expect(DataStoreType.Network).to.not.be.undefined;
+  });
+
+  it('should have type Sync', () => {
+    expect(DataStoreType.Sync).to.not.be.undefined;
+  });
+});
+
+describe('DataStore', () => {
+  describe('constructor', () => {
+    it('should throw and error if a collection is not defined', () => {
+      expect(() => new DataStore()).to.throw();
```

Comment:
We should check the thrown error and make sure it's the error we expect. This way we avoid false positives when something else throws a different error.